integration/utils/templates/create-and-setup-node.cdc

```diff
@@ -1,7 +1,7 @@
 import Crypto
 import FungibleToken from 0xFUNGIBLETOKENADDRESS
 import FlowToken from 0xFLOWTOKENADDRESS
-import FlowIDTableStaking from 0xIDENTITYTABLEADDRESS
+import FlowIDTableStaking from "FlowIDTableStaking"
```

Comment:
Can the other `0xFOO` "addresses" in the other imports also be replaced to the string literal?

Reply:
yeah, I'm working on it. I won't be able to get to it in this PR, but maybe in the next one